src/pages/components/accordion/usage.mdx

```diff
@@ -8,7 +8,7 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 <PageDescription>
 
-An accordion is a vertically stacked list of headers that reveal or hide
+THE accordion is a vertically stacked list of headers that reveal or hide
```

Comment:
should this be all caps? 👀

Reply:
```suggestion
The accordion is a vertically stacked list of headers that reveal or hide
```

Reply:
LOL I was testing something..don't know how it picked up that change